server/routes/overview/overviewController.test.ts

```diff
@@ -18,8 +18,13 @@ import aValidPrisonerSummary from '../../testsupport/prisonerSummaryTestDataBuil
 import { aValidShortQuestionSetEducationAndTraining } from '../../testsupport/educationAndTrainingTestDataBuilder'
 import TimelineService from '../../services/timelineService'
 import aValidTimeline from '../../testsupport/timelineTestDataBuilder'
+import filterTimelineEvents from '../timelineResolver'
```

Comment:
The `timelineResolver` is the main change in this PR to fix the bug. Because of the scope of change I made in `timelineResolver` the tests in `overviewController.test.ts` failed because they were using the real implementation. I have changed this for a mock in this test class. (`timelineResolver` is tested via its own unit test class)